tools/convert/main.cpp

```diff
@@ -91,10 +91,11 @@ static std::string GetOutputFileName(const gfxrecon::util::ArgumentParser& arg_p
     else
     {
         output_filename   = input_filename;
-        size_t suffix_pos = output_filename.find(GFXRECON_FILE_EXTENSION);
-        if (suffix_pos != std::string::npos)
+        auto ext_pos      = output_filename.find_last_of(".");
+        auto path_sep_pos = output_filename.find_last_of(gfxrecon::util::filepath::kPathSepStr);
+        if (ext_pos != std::string::npos && ext_pos > path_sep_pos)
```

Comment:
If there's no path separator in the filename, like I run`gfxrecon-convert gfxrecon_capture_20230601T134916.gfxr`, this `if` will fail because `path_sep_pos` will be `npos` and on MSVC, gcc, and clang that's 18446744073709551615.  I get `gfxrecon_capture_20230601T134916.gfxr.json` but the function should return `gfxrecon_capture_20230601T134916.json` (no ".gfxr")

Reply:
fixed